cli/agent.go

```diff
@@ -417,6 +420,13 @@ func (r *RootCmd) workspaceAgent() *serpent.Command {
 			Default:     "",
 			Value:       serpent.StringOf(&slogStackdriverPath),
 		},
+		{
+			Flag:        "block-file-transfer",
+			Default:     "false",
+			Env:         "CODER_BLOCK_FILE_TRANSFER",
+			Description: "Block file transfer using known applications: nc, rsync, scp, sftp.",
```

Comment:
Perhaps we should define a list of blocked programs to keep the docs and the implementation aligned.